front/src/applications/operationalStudies/views/Scenario/components/SimulationResults.tsx

```diff
@@ -198,115 +197,106 @@ const SimulationResults = ({
   return (
     <div className="simulation-results">
       {/* SIMULATION : SPACE TIME CHART */}
-      {activeBoards.has('std') && projectionData && projectionData.projectedTrains.length > 0 && (
-        <ResizableSection
+
+      {projectionData && (
+        <BoardWrapper
+          name={t('simulationResults.timeSpaceChart')}
+          items={[
+            {
+              title: t('simulationResults.manchetteSettings.waypointsVisibility'),
+              icon: <Eye />,
+              onClick: () => {
+                setWaypointsPanelIsOpen(true);
+              },
+            },
+          ]}
+          hidden={!activeBoards.has('std') || projectionData.projectedTrains.length === 0}
+          resizable
           height={manchetteWithSpaceTimeChartHeight}
           setHeight={setManchetteWithSpaceTimeChartHeight}
           minHeight={MANCHETTE_WITH_SPACE_TIME_CHART_DEFAULT_HEIGHT}
         >
-          <BoardWrapper
-            name={t('simulationResults.timeSpaceChart')}
-            items={[
-              {
-                title: t('simulationResults.manchetteSettings.waypointsVisibility'),
-                icon: <Eye />,
-                onClick: () => {
-                  setWaypointsPanelIsOpen(true);
-                },
-              },
-            ]}
-          >
-            <div className="std-container">
-              <div
-                className="simulation-warped-map d-flex flex-row align-items-stretch"
-                style={{ height: manchetteWithSpaceTimeChartHeight - HIDDEN_CHART_TOP_HEIGHT }}
+          <div className="std-container">
+            <div
+              className="simulation-warped-map d-flex flex-row align-items-stretch"
+              style={{ height: manchetteWithSpaceTimeChartHeight - BOARD_WRAPPER_HEADER_HEIGHT }}
```

Comment:
Hum what about adding this `BOARD_WRAPPER_HEADER_HEIGHT` in the `BoardWrapper` directly ? Since it's related to the board header, maybe we don't need to handle it here